server/gitrest/packages/gitrest-base/src/routes/summaries.ts

```diff
@@ -103,7 +106,7 @@ async function getSummary(
         });
     }
 
-    return wholeSummaryManager.readSummary(sha);
+    return fullSummary;
```

Comment:
😮 big oops. Thanks for the improvement!!

Reply:
Not sure if it was me or you who added that line of code, but I at least reviewed the PR so I also missed it. If anything, I think that might not have been such a bottleneck because we have been persisting the full summary anyway, so often times we would not even get to that line of code. And then we have caching in Historian also. But still good to make this simple change here 😄